### ISSUE TITLE
[Interface-name] Always prefix return error for II18nConfig interface

### ISSUE THREAD

**Issue by artberri** [palantir/tslint#4665]
### Bug Report

- __TSLint version__: 5.15.0
- __TypeScript version__: 3.3.3333
- __Running TSLint via__: gulp-tslint 

#### TypeScript code being linted

```ts
export interface II18nConfig {
    defaultLang: string;
    storageKey: string;
    languages: string[];
    dataStorageKey: string;
    dataLanguages: string[];
}
```

with `tslint.json` configuration:

```json
{
    "rules": {
        "interface-name": [
            true,
            "always-prefix"
        ]
    }
}
```

#### Actual behavior

Throws error: `interface-name  interface name must start with a capitalized I`

#### Expected behavior

No error, II18n... should be an allowed word.

#### Related issue

[Interface-name] Never prefix return error for I18nInterface  https://github.com/palantir/tslint/issues/4476

**Comment by adidahiya** [comment]
Duplicate of #4476
